main/Accordion/README.md

```diff
@@ -37,70 +32,44 @@ yarn add @dooboo-ui/core
 
   ```javascript
   state = {
-    contents: [
+    Data = [
       {
-        title: 'Title 1',
-        body: 'Hi. I love this component. What do you think?',
+        itemTitle: '커스텀 패널',
+        itemBodies: [
+          '패널 커스텀',
+          '패널 커스텀',
+          '패널 커스텀',
+        ],
       },
       {
-        title: 'See this one too',
-        body: 'Yes. You can have more items.',
+        itemTitle: '판매 관리',
+        itemBodies: [
+          '판매처 관리',
+          '공급처 관리',
+          '상품 관리',
+          '재고 관리',
+        ],
       },
       {
-        title: 'Thrid thing',
-        body:
-          'What about very long text? What about very long text? What about very long text? What about very long text? What about very long text? What about very long text? What about very long text? What about very long text? What about very long text? What about very long text? What about very long text? What about very long text?',
+        itemTitle: '주문 배송 관리',
+        itemBodies: [
+          '주문 보기',
```

Comment:
@DevYuns I hope you to use `English` for providing example.

Reply:
Ok. I will replace it.